lib/SiftResponse.php

```diff
@@ -36,6 +42,9 @@ public function __construct($result, $httpStatusCode, $request) {
                 if (array_key_exists('error_message', $this->body)) {
                     $this->apiErrorMessage = $this->body['error_message'];
                 }
+                if (array_key_exists('error', $this->body)) {
```

Comment:
`/v3` endpoints use the `error` field. Should we also add a `description` field?

```
curl https://api.sift.com/v3/does-not-exist
{"error":"not_found","description":"The requested resource does not exist"}
```

Reply:
I'm leaning towards creating `apiError` and `apiDescription` fields to match the union of `/v3` and `/v2xx` error response fields, and `curlError` for the curl error.